kubernetes/configs/dev/commons.sh

```diff
@@ -42,5 +42,6 @@ PRIVACY_NOTICES_DYNAMO_TABLE_NAME="interop-privacy-notices-dev"
 PRIVACY_NOTICES_ACCEPTANCE_DYNAMO_TABLE_NAME="interop-privacy-notices-acceptances-dev"
 PRIVACY_NOTICES_UPDATER_PRIVACY_POLICY_UUID="0df21ff6-3e8f-4320-af8f-23dea9135d57"
 PRIVACY_NOTICES_UPDATER_TERMS_OF_SERVICE_UUID="6bf8412a-41a7-41a0-82dc-26286ce61b1a"
+PRIVACY_NOTICES_CONTAINER=""
```

Comment:
@beetlecrunch do we have this too?

Reply:
copied from [here](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/296/files#diff-e74e70e12a4efa4ed611fd20d4a8734c8b4f276447c4d53f04e155055c3b334bR5)